src/pybind/mgr/cephadm/ssh.py

```diff
@@ -215,6 +215,7 @@ def check_execute_command(self,
 
     async def _write_remote_file(self,
                                  host: str,
+                                 unique_id: str,
```

Comment:
Do we ever actually have plans to use something other than the fsid here? If not, it seems like we could just have this function find the fsid itself rather than needing to pass in an fsid every time.